### DISCUSSION TITLE
/permtrust command

### DISCUSSION THREAD

**Discussion by QarthO** [GriefPrevention/GriefPrevention#2054] [Ideas]
If Notch would type ``/permtrust QarthO`` then  I would be permanently trusted to all of Notch's current AND future claims.

Currently its super frustrating how if you're playing in a large group, how every single time you make a new claim, you have to trust everyone again. 

Would also add ``/permcontainertrust`` and ``/permaccesstrust`` with the same concept

Edit: I will need to brainstorm new name. Perm trust is not supposed to be the same as permission trust.
Perm is short for permanent trust.
Maybe ``/futuretrust``? Idk

**Comment by daretmavi** [comment]
This is exactly what my son asked for.

**Reply by QarthO** [reply]
Don't worry! I made a plugin that automatically makes every trust work on future claims.

It's called qGPTrust. Just add it in your plugins folder in addition to GriefPrevention. No other setup required

https://www.spigotmc.org/resources/qgptrust.109829/

I'm hoping after GriefPrevention adds this feature I'll be able to remove my plugin. I'm also willing to code for them if needed!
